src/main/java/sirius/biz/tenants/Tenants.java

```diff
@@ -284,4 +284,60 @@ public String fetchCachedTenantName(I tenantId) {
     public String fetchCachedTenantName(BaseEntityRef<I, T> tenantRef) {
         return fetchCachedTenant(tenantRef).map(tenant -> tenant.getTenantData().getName()).orElse("");
     }
+
+    /**
+     * Provides access to the user account with the given id
+     * <p>
+     * This utilizes the cache maintained by the {@link TenantUserManager} and is therefore quite efficient
+     *
+     * @param userId the id of the user to fetch
+     * @return the user account with the given id or an empty optional if the user cannot be resolved
+     */
+    public Optional<U> fetchCachedUserAccount(I userId) {
+        if (Strings.isEmpty(userId)) {
+            return Optional.empty();
+        }
+
+        return Optional.ofNullable(getTenantUserManager().fetchAccount(String.valueOf(userId)));
+    }
+
+    /**
+     * Boilerplate to quickly fetch the email of the user with the given id.
+     *
+     * @param userId the user to fetch the email for
+     * @return the email of the user or an empty string if the user doesn't exist
+     */
+    public String fetchCachedUserMail(I userId) {
```

Comment:
maybe also add a fetchCachedUserName which outputs the person name or UserAccount.toString or the like?